docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -54,35 +54,25 @@ helm get manifest demo | grep size
 .
 .
   size: 1
-      initial_heap_size: "800M"
-      max_heap_size: "800M"
-    :[{\"expr\":\"sum(mcac_table_memtable_off_heap_size{cluster=~\\\"$cluster\\\"\
-    :\"A\"},{\"expr\":\"sum(mcac_table_memtable_on_heap_size{cluster=~\\\"$cluster\\\
-    ,\"description\":\"Total sizes of the data on distinct nodes\",\"fill\":0,\"gridPos\"\
-    datasource\":\"$PROMETHEUS_DS\",\"description\":\"Maximum JVM Heap Memory size\
-    \ (worst node) and minimum available heap size\",\"fill\":1,\"gridPos\":{},\"\
 ```
 
-Notice the value of `size: 1` from cassdc.yaml. This is the Cassandra DataCenter definition. 
+The value of `size: 1` is from cassdc.yaml, which is the Cassandra DataCenter definition. 
 
-To scale up, you could change the `size` to 3. Example with helm:
+To scale up, you could change the `size` to 3. Example:
 
 ```bash
-helm upgrade demo k8ssandra/k8ssandra --set k8ssandra.size=3 --reuse-values
+helm upgrade demo k8ssandra/k8ssandra --set cassandra.datacenters\[0\].size=3,cassandra.datacenters\[0\].name=dc1
```

Comment:
I think it was @burmanm that was mentioning this recently, but because of the way the setting of the arrays on the helm CLI work you need to set any values under that item each time - you've done it here setting name in addition to size but might make sense to explain that here in addition to the example.  The example looks good now though.

Reply:
Ah ok - thanks.

Reply:
Updated